lib/form/gui.flow

```diff
@@ -490,6 +492,42 @@ Lines(rows) {
 	else Grid(map(rows, v2a));
 }
 
+// Helper for supporting RTL in Lines variations
+linesToLinesA(ls : [Form]) -> [Form] {
+	if (getDefaultRtl()) linesToLinesRTL(ls) else ls;
+}
+
+linesToLinesRTL(ls : [Form]) -> [Form] {
+	widths = map(ls, \l -> make(0.0));
+	maxWidthB = arrayMaxu(widths);
+
+	mapi(
+		ls,
+		\i, l -> {
+			diffWidth = make(0.0);
+
+			Constructor(
+				Translate(
+					diffWidth,
+					make(0.0),
+					Inspect([Width(widths[i])], l)
+				),
+				\ -> {
+					uns = connectSelect2Distinctu(maxWidthB.first, widths[i], diffWidth, \maxWidth, width -> maxWidth - width);
+					\ -> {
+						if (i == 0) maxWidthB.second();
```

Comment:
only if i == 0?

Reply:
maxWidthB is "global" behavior, we should destroy it only once.
there is another option of course, I could pass it to the Lines level and wrap resulting Lines with another Constructor.

Reply:
rewritten to be less obstructive